olive/systems/azureml/aml_system.py

```diff
@@ -718,6 +702,20 @@ def _create_metric_component(
         # metric component
         return cmd(**args)
 
+    def copy_code(self, code_files: List, code_root: Path):
```

Comment:
nit: maybe rename `code_root` to `targer_code_path` or something else?

Reply:
will do